pallets/attestation/src/mock.rs

```diff
@@ -167,8 +167,8 @@ pub fn insert_attestation<T: Config>(claim_hash: ClaimHashOf<T>, details: Attest
 	}
 }
 
-pub fn sr25519_did_from_seed(seed: &[u8; 32]) -> SubjectId {
-	MultiSigner::from(sr25519::Public(*seed)).into_account().into()
+pub fn sr25519_did_from_seed(public_key: &[u8; 32]) -> SubjectId {
```

Comment:
At this point 😃

```suggestion
pub fn sr25519_did_from_public_key(public_key: &[u8; 32]) -> SubjectId {
```

Reply:
[a375f3f](https://github.com/KILTprotocol/kilt-node/pull/540/commits/a375f3f7d6a6958503d76b34c71dfca3af4090e4)